Library/Formula/pcap_dnsproxy.rb

```diff
@@ -0,0 +1,63 @@
+class PcapDnsproxy < Formula
+  desc "A powerful DNS proxy designed to anti DNS spoofing"
+  homepage "https://github.com/chengr28/Pcap_DNSProxy"
+  url "https://github.com/chengr28/Pcap_DNSProxy/archive/v0.4.2.tar.gz"
+  sha256 "0737b16283e84bc94f63066f448ba84416141b8d5a86b5e01f1f97164d9d44f2"
+  head "https://github.com/chengr28/Pcap_DNSProxy.git"
+
+  depends_on :xcode => :build
+  depends_on "libsodium" => :build
+
+  def install
+    rm Dir[buildpath/"Source/LibSodium/*"]
+    ln_s "#{Formula["libsodium"].opt_lib}/libsodium.a", "#{buildpath}/Source/LibSodium/LibSodium_Mac.a"
+    ln_s Dir[Formula["libsodium"].opt_include/"*"], "#{buildpath}/Source/LibSodium/"
+    xcodebuild "-project", "./Source/KeyPairGenerator.xcodeproj", "-target", "KeyPairGenerator", "-configuration", "Release", "SYMROOT=build"
+    xcodebuild "-project", "./Source/Pcap_DNSProxy.xcodeproj", "-target", "Pcap_DNSProxy", "-configuration", "Release", "SYMROOT=build"
+
+    bin.install "Source/build/Release/KeyPairGenerator"
+    bin.install "Source/build/Release/Pcap_DNSProxy"
+    (etc/"pcap_DNSproxy").install Dir["Source/ExampleConfig/*.{ini,txt}"]
+  end
+
+  test do
+    mkdir testpath/"pcap_dnsproxy"
+    cp Dir[etc/"pcap_dnsproxy/*"], testpath/"pcap_dnsproxy/"
+    inreplace testpath/"pcap_dnsproxy/Config.ini" do |s|
+      s.gsub! /Hosts Only.*/, "Hosts Only = 1"
+      s.gsub! /Listen Port.*/, "Listen Port = 9999"
+    end
+    pid = fork do
+        exec bin/"Pcap_DNSProxy", "-c", testpath/"pcap_dnsproxy/", [:out, :err]=>"/dev/null"
+    end
+    system "dig", "google.com", "@127.0.0.1", "-p", "9999", "+short"
+    Process.kill "TERM", pid
```

Comment:
```
pid = fork { system bin/"Pcap_DNSProxy", "-c", testpath/"pcap_dnsproxy/" }
...
Process.kill 9, pid
```
Can't kill the Pcap_DNSProxy's process, I use:
`pid = fork { exec bin/"Pcap_DNSProxy", "-c", testpath/"pcap_dnsproxy/", [:out, :err]=>"/dev/null" }`
instead.
The new code can ensure the Pcap_DNSProxy killed whether `dig` succeed or not.